res/menu/upload_task_menu.xml

```diff
@@ -13,6 +13,10 @@
         android:visible="true"
         android:title="@string/task_remove" />
 
+    <item android:id="@+id/remove_all_cancelled"
```

Comment:
`upload_task_menu` and `download_task_menu` are exactly the same, remove the two, use `transfer_task_menu` instead.